### DISCUSSION TITLE
(CLI) : Issues with API route on fresh CLI's init/start

### DISCUSSION THREAD

**Discussion by mtimour** [supabase/supabase#4497] [Questions] [answered]
Brand new local deployment using CLI on Windows via WSL2.  
This is what `supabase start ` prints:
```
Started local development setup.

         API URL: http://localhost:54321
          DB URL: postgresql://postgres:postgres@localhost:54322/postgres
      Studio URL: http://localhost:54323
        anon key: <....>
service_role key: <...>
```

The API page fails with the message "Something went wrong ... Error: [500] An error has occured: NetworkError when attempting to fetch resource.".  

Browser dev tools show that Studio tries to access `localhost:8000` which is strange because CLI starts all containers on 5432X ports. 

![image](https://user-images.githubusercontent.com/29662132/146227314-ac093030-c5b9-43fd-a4fb-f71b708fafdf.png)

**Does anyone have the same issue ?**

**Comment by gabydd** [comment, marked as answer]
The localhost:8000 is hard-coded [here](https://github.com/supabase/supabase/blob/master/studio/pages/api/props/project/[ref]/api.ts#L45) which needs to be changed to be based of the env instead

**Reply by mtimour** [reply]
Thanks for the clarification.  I will try to change CLI's config for API_URL to be 8000 for now. Docker deployment works fine because by default it is 8000 for kong.
